include/service/entities/custom_object_service.js

```diff
@@ -361,9 +361,10 @@ module.exports = function CustomObjectServiceModule(pb) {
 
         //make sure we have the type for the object passed in
         getCustObjType(custObjType, function(err, custObjType) {
-            if (util.isError(err)) {
+            if (util.isError(err) || util.isNullOrUndefined(custObjType)) {
```

Comment:
Calling back with only an empty error would be incorrect behavior here.  There should be a condition such that:
```
else if (util.isNullOrUndefined(custObjType)) {
  return cb(new Error('An invalid custom object type: ' + custObjType + ' was found.'));
}
```